.github/workflows/ci.yml

```diff
@@ -39,32 +39,36 @@ jobs:
           - java: 16
             os: ubuntu-latest
     runs-on: ${{ matrix.os }}
+    if: (github.repository == 'apache/shenyu')
     steps:
       - name: Support longpaths
         if: ${{ matrix.os == 'windows-latest'}}
         run: git config --system core.longpaths true
       - uses: actions/checkout@v2
         with:
           submodules: true
-      - name: Set Skip Env Var
-        uses: ./.github/actions/skip-ci
+      - uses: dorny/paths-filter@v2
```

Comment:
It may be wrong to use third-party actions directly in the apache repository. We can add git submodules in `./.github/actions`. For example: https://github.com/apache/shenyu-helm-chart/tree/main/.github/actions

Reply:
Is there an official explanation?

Reply:
<img width="1031" alt="image" src="https://user-images.githubusercontent.com/62384022/187452827-a8189678-3c9f-4f85-986d-889689daa766.png">
This is the error we encountered before. But it looks ok in your pr's ci 🤔, maybe we can try it first.

Reply:
em. This is a special case. If the action is generic and multiple projects are using it, we can try to request `Apache ` for permission to use it. : )